README.md

```diff
@@ -27,8 +27,22 @@ Eclipse Theia is an extensible platform to develop full-fledged multi-language C
 "Theia" is a trademark of the Eclipse Foundation
 https://www.eclipse.org/theia
 
+## What is this?
+
+Eclipse Theia Blueprint product is a **template** to showcase the capabilities of Theia as well as how to build desktop-based products based on the platform. Theia Blueprint assembles a selected subset of existing Theia features and extensions. We provide installers for Theia Blueprint to be downloaded (see below) as well as documentation on how to customize this template to build a product and installers for your own Theia-based product.
```

Comment:
> based on the platform

hello,
just fyi: some time ago, we've agreed on using _framework_ instead of _platform_. See https://github.com/eclipse-theia/theia/issues/7519

Reply:
Thanks, we also have some feedback about the documentation wording here: https://github.com/eclipse-theia/theia-website/pull/133
We will address this together then once it's clear where to put the documentation

Reply:
Resolving as we will align the wording after its available/accepted on the documentation page